### PR TITLE
Video storage

### PR DESCRIPTION
**Pull request by markulrich**
- Serve all videos from GC storage
- For RequestInvite, upload to GC storage and then show user a preview of what has been uploaded

For privacy reasons, do not directly serve what user uploaded to others. Must first strip all metadata (especially location) from the video, which will currently be an offline workflow.

### CODE REVIEW

File: src/components/InviteVideo.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+interface Props {
+  readonly userId: string;
+  readonly memberId: string;
+}
+
+export default class InviteVideo extends React.Component<Props, {}> {
+  public render() {
+    // TODO group by uid instead of mid?
+    const videoUrl = `https://storage.googleapis.com/raha-video/${this.props.memberId}/invite.mp4`
```
Comment (markulrich) [inline src/components/InviteVideo.tsx]: @rahulgi after update this will change to props.userId ?

Comment (rahulgi) [reply src/components/InviteVideo.tsx]: currently sticking with mid

File: src/components/RequestInvite.tsx [hunk 2]
```diff
@@ -8,14 +8,22 @@ import {
...
 import { AppState } from "../store";
+import InviteVideo from "./InviteVideo";
+import Loading from "./Loading";
 import LogIn from "./LogIn";
-import YoutubeVideo, { getYoutubeUrlVideoId } from "./YoutubeVideo";
+import Video from "./Video";
+import VideoUploader from "./VideoUploader";
 
 const RequestInviteElem = styled.main`
+  div {
```
Comment (osdiab) [inline src/components/RequestInvite.tsx]: this is a concerningly general style lol

File: src/components/RequestInvite.tsx [hunk 3]
```diff
@@ -90,39 +95,60 @@ export class RequestInvite extends React.Component<Props, State> {
     this.setState({ videoUrl: event.currentTarget.value });
   };
 
-  private readonly setFullName = (
+  private readonly setauthFullName = (
```
Comment (osdiab) [inline src/components/RequestInvite.tsx]: capitalization typo?

**Pull request merged**